crates/codegen/src/get_node_properties.rs

```diff
@@ -439,6 +439,17 @@ fn custom_handlers(node: &Node) -> TokenStream {
                 tokens.push(TokenProperty::from(Token::Or));
                 tokens.push(TokenProperty::from(Token::Replace));
             }
+            if let Some(n) = &n.view {
+                match n.relpersistence.as_str() {
```

Comment:
Here, I would expect `Temporary` token to be part of `RangeVar` node but parser still panics so I had to make it part of `ViewStmt`, very similar to what we had with the `As` token being part of `CreateDomainStmt` in #61

cc @psteinroe

Reply:
thats an interesting case! I agree with your reasoning. to give a bit of context. Take the substring `create temporary view comedies` as an example. The `create` and the `view` token should be part of the `ViewStmt` itself, while the `temporary` is definitely part of the `RangeVar` node. We cannot create a valid tree out of this, since we would have to open `RangeVar` at `temporary`, close it to go back to `ViewStmt ` and then open it again to be back at `RangeVar` for `comedies`. So yes, we should pull `temporary` back up into `ViewStmt`. thanks!

Reply:
it makes sense, thanks for the clarification